minio/helpers.py

```diff
@@ -137,20 +135,16 @@ def parts_manager(data, part_size=5*1024*1024):
     :param part_size: Individual part number defaults to 5MB.
     :return: Returns :class:`PartMetadata <PartMetadata>`
     """
-    tmpdata = io.BytesIO()
-    md5hasher = Hasher.md5()
-    sha256hasher = Hasher.sha256()
-    total_read = 0
-    while total_read < part_size:
+    part_data = io.BytesIO()
```

Comment:
Can you check if part_size % 1024 == 0 and throw an error otherwise ? because we are reading with 1024 bytes chunk and this algorithm won't be correct if we change that read chunk size